### PR TITLE
Message when find-mentor page doesn't have mentor to display

### PR DESCRIPTION
**Pull request by SylviaZiyuZhang**

### CODE REVIEW

File: src/main/webapp/static/scripts/find-mentor.js [hunk 1]
```diff
@@ -88,7 +88,16 @@ for (const buttonName in buttonDict) {
       if (mentorCardContainer.childElementCount < MIN_MENTOR_THRESHOLD) {
         refillMentors();
       }
+
       getActiveMentorCard();
     }
   });
 }
+
+const updateButtonContainer = () => {
```
Comment (muditg317) [inline src/main/webapp/static/scripts/find-mentor.js:97]: maybe call this method every time a rating is sent. that way, once they run out, they get the same update. also maybe toggle visibility of this div and reveal a second div witht he warning. that way we can bring the buttons back if something changes

File: src/main/resources/templates/find-mentor.html [hunk 2]
```diff
@@ -33,5 +34,5 @@
 
 {% block scripts %}
 {{ super() }}
-<script charset="utf-8" type="text/javascript" src="static/scripts/find-mentor.js"></script>
+<script charset="utf-8" type="text/javascript" src="static/scripts/find-mentor.js" onload="updateButtonContainer()"></script>
```
Comment (muditg317) [inline src/main/resources/templates/find-mentor.html:37]: you can also just call the method in the js file. similar to line 69: refillMentors();

**Review by muditg317 — COMMENTED** [review]
i left some suggestions

File: src/main/webapp/static/scripts/find-mentor.js (hunk 1 above)
Comment (SylviaZiyuZhang) [reply src/main/webapp/static/scripts/find-mentor.js:97]: This function is called onload so the buttons should come back when stuff changes and mentors appear. Calling this method every time a rating is sent doesn't work if the user refreshes the page after seeing the message (the button reappears, which is not intended).

**Review by sagunb — APPROVED** [review]

**Pull request merged**